lib/ui/screen-components/protected/restaurant(vendor)/dashboard/restaurant-stats-table/index.tsx

```
@@ -0,0 +1,107 @@
+                      //   loading={salesDetailsLoading}
+                      //   title={item._type}
+                      //   data={item.data}
+                      //   amountConfig={{ currency: CURRENCY_CODE ?? 'USD' }}
+                      // />
+                      <>{index}</>
+                    );
+                  })
+                )}
+              </div>
+            </div>
+            <Divider className="my-4 border-t border-gray-300" />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)